src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -105,6 +105,14 @@ private void DrawSettingsEditorWindow()
             {
                 DrawEndpoints(so);
             }
+
+            GUILayout.Space(5);
+            _showSwitch = EditorGUILayout.Foldout(_showSwitch, "Nintendo Switch (Requires Nintendo Switch Bugsnag plugin)", true);
```

Comment:
I imagined this in a tooltip is that not possible on a group header?

Reply:
Done 👍